modules/lm-coursier/src/main/scala/lmcoursier/internal/ResolutionRun.scala

```diff
@@ -87,10 +90,10 @@ object ResolutionRun {
         .withResolutionParams(
           params
             .params
-            .addForceVersion((if (isCompileConfig) params.interProjectDependencies.map(_.moduleVersion) else Nil): _*)
-            .withForceScalaVersion(isCompileOrScalaToolConfig && params.autoScalaLibOpt.nonEmpty)
+            .addForceVersion((if (isSandboxConfig) Nil else params.interProjectDependencies.map(_.moduleVersion)): _*)
+            .withForceScalaVersion(params.autoScalaLibOpt.nonEmpty)
```

Comment:
Maybe for future reference: the stricter check (forcing the version only in the the compile or scala-tool configs) may have been done for a reason. I can't recall which though, and this didn't break any tests, so let's merge this for now…